srcpkgs/noisetorch/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'noisetorch'
+pkgname=noisetorch
+version=0.11.3
+revision=1
+archs="x86_64*"
+wrksrc=NoiseTorch-${version}
+build_style=gnu-makefile
+hostmakedepends="go"
+depends="polkit"
+short_desc="Real-time microphone noise suppression on Linux"
+maintainer="Soc Virnyl S. Estela <renegan.ronin@gmail.com>"
+license="GPL-3.0-or-later"
```

Comment:
While the project is release under GPL version 3, it's not clear whether they intend for it to be released under later versions of the license as well. Usually that is included in the README or source files. I couldn't find such labels, however. Furthermore, the `untar.go` file included might be licensed differently. From `untar.go`:
```
/*Copyright (c) 2009 The Go Authors. All rights reserved.Redistribution and use in source and binary forms, with or withoutmodification, are permitted provided that the following conditions aremet:   * Redistributions of source code must retain the above copyrightnotice, this list of conditions and the following disclaimer.   * Redistributions in binary form must reproduce the abovecopyright notice, this list of conditions and the following disclaimerin the documentation and/or other materials provided with thedistribution.   * Neither the name of Google Inc. nor the names of itscontributors may be used to endorse or promote products derived fromthis software without specific prior written permission.THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOTLIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FORA PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHTOWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOTLIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANYTHEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USEOF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.*/
// This is yoinked from https://github.com/golang/build/blob/master/internal/untar/untar.go// which is unfortunately an internal package we can't import, but it's exactly what we need.// So just copy paste it.
```
They just... "copy paste it." Seems odd.

Reply:
I guess I should ask the project maintainers about this. Seems odd indeed.

Reply:
That is a BSD 3 Clause license, I am pretty sure you don't have to care about it.

Reply:
You're right - still not sure if GPL-3.0-only or GPL-3.0-or-later

Reply:
Ugh it has an additional clause on top of GLP 3: https://github.com/lawl/NoiseTorch/blob/master/LICENSE#L5

Using https://github.com/lawl/NoiseTorch/blob/master/c/ladspa/module.c as the most restrictive file, the final package should be GPL-3.0-or-later. No idea how to mention the additional clause.,

Reply:
Rust has a similar restriction (not mentioned in the license) and we don't mention it in the template.